### DISCUSSION TITLE
after years working niow I have a problem

### DISCUSSION THREAD

**Discussion by abueloedu24** [aclap-dev/video-downloadhelper#433] [Help and Q&A]
### Your question

I use net.downloader for many years, since I have this old Mac - with OS Sierra 10.12 - from 2014. I had the CoApp but probably I clicked remove or something and is asking for it again. I looked on the Apps and I have VhCoApp = = downloadhelper.coapp 16.3.dmg -modified May 25, 2021 - Browser I mainly use Firefox Mozilla. 
I had an open heart surgery and my memory fluctuates. I am retired and old. I AM ASKING for help, how to download the
proper CoAp and I remember paying a few bucks. I would pay the 25 or whatever is now. 

### Addon & CoApp version and details

I use net.downloader for many years, since I have this old Mac - with OS Sierra 10.12 - from 2014. I had the CoApp but probably I clicked remove or something and is asking for it again. I looked on the Apps and I have VhCoApp = = downloadhelper.coapp 16.3.dmg -modified May 25, 2021 - Browser I mainly use Firefox Mozilla. 
I had an open heart surgery and my memory fluctuates. I am retired and old. I AM ASKING for help, how to download the
proper CoAp and I remember paying a few bucks. I would pay the 25 or whatever is now.

**Comment by jcfrog** [comment]
Hi, I'm afraid there is no more solution for OS Sierra 10.12
For the license please send us an email at licenses@downloadhelper.net
